Source/SwiftLintFramework/Rules/Style/LetVarWhitespaceRule.swift

```diff
@@ -225,15 +225,43 @@ private extension SwiftDeclarationKind {
 }
 
 private extension File {
-    // Zero-based line number for the given a byte offset
     func line(byteOffset: Int, startFrom: Int = 0) -> Int {
-        for index in startFrom..<lines.count {
+        return lineFast(byteOffset: byteOffset, startFrom: startFrom)
```

Comment:
what if we use `contents.bridge().lineAndCharacter(forByteOffset: byteOffset)?.line).map { $0 - 1 }`? 

This does a binary search as well.

Reply:
It seems to be working as well.